src/pages/index.js

```diff
@@ -96,6 +227,41 @@ profileAddBtn.addEventListener("click", () => {
   addCardPopup.open();
 });
 
+//Edit Avatar Form
+profileAvatarBtn.addEventListener("click", () => {
+  editAvatarPopup.open();
+});
+
+profileAvatarSubmitBtn.addEventListener("submit", handleAvatarEditSubmit);
```

Comment:
`profileAvatarSubmitBtn.addEventListener("submit", handleAvatarEditSubmit);
`

Please, delete it


Any submit handler should be added via `PopupWithForm` . You have `editAvatarPopup` for that